SU2_PY/SU2_Nastran/TestCase/FlatPlateModel.m

```diff
@@ -0,0 +1,157 @@
+%% This script uses the theodorsen theory for the flutter of a flat plate
+% Author: Giampiero Bindolino, Nicola Fonzi, Vittorio Cavalieri
+%%% Structural definition
+clear all;close all;clc
```

Comment:
I don't know if this is used in any of your examples, but any code file in SU2 should carry the SU2 header and licence notice, otherwise technically it is proprietary code.